openvdb_houdini/openvdb_houdini/SOP_OpenVDB_Sample_Points.cc

```diff
@@ -707,14 +706,16 @@ SOP_OpenVDB_Sample_Points::Cache::cookVDBSop(OP_Context& context)
         }
 
         // timing: end time
-        tbb::tick_count time_end = tbb::tick_count::now();
+        auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(
```

Comment:
Minor nitpick here is that we only need `seconds` when `if (verbose)` so we can avoid the extra `now()` call in the non-verbose case.

Reply:
I have fixed this egregious mistake